CHANGES.md

```diff
@@ -4,6 +4,10 @@
 
 (nothing yet)
 
+## 2.0.3
+
+* #173: support content-md5 header generated from different node versions
```

Comment:
I'd fill this out a little bit, mentioning at least the new option name. Could link to the new doc section.

Reply:
Okay - added.